src/sql/src/plan/query.rs

```diff
@@ -4219,7 +4219,10 @@ fn plan_aggregate(
             if args.is_empty() {
                 sql_bail!(
                     "{}(*) must be used to call a parameterless aggregate function",
-                    name
+                    ecx.qcx
+                        .scx
+                        .humanize_resolved_name(name)
+                        .expect("name actually resolved")
```

Comment:
Why do we know `name` will always resolve? Is that guaranteed by some caller?

Reply:
This is guaranteed by the new name resolution pass over functions.